doc/stdlib/io-network.rst

```diff
@@ -624,7 +624,7 @@ Text I/O
 
    The columns are assumed to be separated by one or more whitespaces. The end of line delimiter is taken as ``\n``\ . If all data is numeric, the result will be a numeric array. If some elements cannot be parsed as numbers, a heterogeneous array of numbers and strings is returned.
 
-.. function:: writedlm(f, A, delim='\\t')
+.. function:: writedlm(f, A, dl='\\t'; opts)
```

Comment:
yeah I think doc consistency will be more visible than code naming consistency. we could change the latter too if we feel like it.